src/main/java/com/mathworks/ci/MatlabRunTestsStepExecution.java

```diff
@@ -68,12 +68,16 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
                     "addpath('" + genScriptLocation.getRemote().replaceAll("'", "''") + "'); ";
             final String matlabScriptName = getValidMatlabFileName(genScriptLocation.getBaseName());
 
-            ProcStarter matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener,
-                    envVars, cmdPrefix + matlabScriptName, uniqueTmpFldrName);
+            ProcStarter matlabLauncher =
+                    getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
+                            cmdPrefix + matlabScriptName + ",delete('.matlab/"
+                                    + genScriptLocation.getBaseName() + "/" + matlabScriptName
+                                    + ".m'),run(runnerScript),rmdir(tmpDir,'s')",
+                            uniqueTmpFldrName);
             
             // prepare temp folder by coping genscript package and writing runner script.
             prepareTmpFldr(genScriptLocation,
-                    getRunnerScript(MatlabBuilderConstants.TEST_RUNNER_SCRIPT, envVars.expand(getCommandArgs())));
+                    getRunnerScript(MatlabBuilderConstants.TEST_RUNNER_SCRIPT, envVars.expand(getCommandArgs()),uniqueTmpFldrName));
```

Comment:
Should we rename uniqueTmpFolderName? To me this sounds like it is in the /tmp folder, not the .matlab folder.

Reply:
Done. changed to uniqueMatlabResourceFldr